src/components/Hero.jsx

```diff
@@ -1,14 +1,14 @@
-import { PersonalizedText } from './shared/typography';
+import { PersonalizedText, Text } from './shared/typography';
 import { HeroText1 } from './shared/typography';
 import { HeroText2 } from './shared/typography';
 import { imgURLs } from '../data/eventsData';
 
 const Hero = () => {
 	return (
-		<div style={{ margin: '0 60px' }}>
-			<div className='flex flex-col md:flex-row'>
-				<div className='w-[100%] md:w-1/2 '>
-					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+		<div className='mx-5 lg:mx-14 xl:max-w-[1483px]'>

```

Comment:
Works for now but use container

Reply:
container won't work it only sets the `max-width` of an element to match the `min-width` of the current breakpoint.